Sources/Harmonize/Core/ScopeBuilder/HarmonizeScopeBuilder.swift

```diff
@@ -39,7 +39,7 @@ internal class HarmonizeScopeBuilder {
         let folderKey = folder ?? "nil"
         let inclusionsKey = includingOnly.joined(separator: ",")
         let exclusionsKey = exclusions.joined(separator: ",")
-        return "\(folderKey)|\(inclusionsKey)|\(exclusionsKey)"
+        return "folder:\(folderKey)|including:\(inclusionsKey)|excluding:\(exclusionsKey)"
```

Comment:
This is unnecessary but it made the key easier for me to understand in the debugger